packages/headless/src/api/platform-client.ts

```diff
@@ -76,7 +82,15 @@ export class PlatformClient {
           shouldRetry && logger.info('Platform retrying request');
           return shouldRetry;
         },
+      }).catch((error: Error) => {
+        if (error.message !== 'Failed to fetch') {
+          throw error;
+        }
+        return PlatformClientErrorResponse.Disconnected;
       });
+      if (typeof response === 'number') {
```

Comment:
I originally did `!(response instanceof Response)`, but the `Response` constructor doesn't exist in unit tests apparently.

Reply:
It's possible to import the `Response` class from the `cross-fetch` library if needed.